### PR TITLE
Add RSKIP298 - Bridge peg-out request index

### PR DESCRIPTION
**Pull request by josedahlquist**

### CODE REVIEW

**Review by kelvinator07 — APPROVED** [review]

File: IPs/RSKIP298.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
+# Bridge peg-out request index
+
+|RSKIP          |298           |
+| :------------ |:-------------|
+|**Title**      |Bridge peg-out request index |
+|**Created**    |18-APR-22 |
+|**Author**     |JD |
+|**Purpose**    |Sca |
```
Comment (nathanieliov) [inline IPs/RSKIP298.md:8]: What means 'Sca'?

**Review by nathanieliov — APPROVED** [review]

File: IPs/RSKIP298.md [hunk 2]
```diff
@@ -0,0 +1,64 @@
+# Bridge peg-out creationindex
```
Comment (marcos-iov) [inline IPs/RSKIP298.md:1]:
```suggestion
# Bridge peg-out creation index
```

**Review by marcos-iov — COMMENTED** [review]
Looks good, I think this can be rebased and changed from Draft to Ready for review

File: README.md [hunk 3]
```diff
@@ -197,6 +197,7 @@ RSK Improvement Proposals
 | 285 |[Utility Methods to Make PPA safer](IPs/RSKIP285.md)|  25-OCT-21 | SDL | Usa,Sec | Core | 1 | Draft |
 | 290 |[Adjust Testnet block minimum difficulty](IPs/RSKIP290.md)|  18-NOV-21 | AE | Usa | Core | 1 | Draft |
 | 291 |[Network Upgrade: Hop](IPs/RSKIP291.md)|  14-DEC-21 | AE | Usa, Sec | Core | 2 | Draft |
+| 298 |[Bridge peg-out request index](IPs/RSKIP298.md)|  18-APR-22 | JD | Sca | Core | 2 | Draft |
```
[the remark this answers is not in the document]
Comment (nathanieliov) [reply README.md]: Done

**Review by marcos-iov — APPROVED** [review]

**Pull request merged**